### PR TITLE
14 User Sees GitHub Following

### PR DESCRIPTION
**Pull request by alexander-mathieu**
This PR adds a GitHub following section on the User's dashboard.

- Adds feature spec for GitHub following section
- Adds following method to UserShowFacade
- Adds following_by_user method to GithubService
- Updates VCR cassette used in User dashboard feature tests
- Adds GithubService following_by_user spec with fixture data

### CODE REVIEW

File: spec/features/user/user_sees_github_repos_on_dashboard_spec.rb [hunk 1]
```diff
@@ -25,20 +25,21 @@
       end
     end
 
-    it 'does not display if user missing a gd token' do
+    it "I do not see a GitHub section if I'm missing a GitHub token" do
```
Comment (BrennanAyers) [inline spec/features/user/user_sees_github_repos_on_dashboard_spec.rb:28]: 👍

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -15,18 +15,26 @@
     <section class='github-dashboard'>
       <h1>GitHub</h1>
       <section class='github-dashboard-repos'>
-        <h2>Repositories</h2>
+        <h3>Repositories</h3>
```
Comment (BrennanAyers) [inline app/views/users/show.html.erb:18]: Good thing we did some Heroku testing!

**Review by BrennanAyers — APPROVED** [review]
Very similar to the /followers request and #followers_by_user, but glad we could clean up a few things along the way.

File: spec/features/user/user_sees_github_repos_on_dashboard_spec.rb (hunk 1 above)
Comment (bplantico) [reply spec/features/user/user_sees_github_repos_on_dashboard_spec.rb:28]: Good refactor to be more specific about what is being tested

**Pull request merged**